### PR TITLE
fix applying tokens mask

### PR DESCRIPTION
**Pull request by rebel-ykchoi**
### 🚀 Summary of Changes

tokens_mask should be added to router_logits to make -inf for dummy tokens, instead of multiplying it.

> *What does this PR do? What feature, fix, or improvement does it bring?*

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] ✨ Feature (`feature`)
* [ ] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [ ] 🛠 Bug fix (`bug-fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

**Comment by rebel-yhboo** [comment]
https://github.com/RBLN-SW/vllm-rbln/blob/7a3aba68d51ebb7f4d06eb9033beb711fafb4709/vllm_rbln/model_executor/layers/fused_moe/layer.py#L275

여기는 저희쪽 module에는 영향이 없는 부분일까요?
